app/src/main/java/org/fossasia/openevent/dbutils/DbContract.java

```diff
@@ -41,15 +41,14 @@ public DbContract() {
         public static final String CREATE_TABLE =
                 "CREATE TABLE " + TABLE_NAME
                         + " ("
-                        + _ID + INT_TYPE + PRIMARY_KEY + COMMA_SEP
-                        + EVENT_ID + INT_TYPE + COMMA_SEP
+                        + VER_ID + INT_TYPE + PRIMARY_KEY + COMMA_SEP
+                        //+ EVENT_ID + INT_TYPE + COMMA_SEP
```

Comment:
Cool I guess this is pretty much finalised schema until some changes are made in server.